main_test.go

```diff
@@ -49,3 +49,7 @@ func TestMain(m *testing.M) {
 
 	os.Exit(ret)
 }
+
+func createEmptyFile(path string) (err error) {
```

Comment:
Q. Could this function be part of a different file? something like common.go or lib.go?